components/server/src/user/user-service.ts

```diff
@@ -248,6 +249,36 @@ export class UserService {
         return AttributionId.render({ kind: "team", teamId: project.teamId });
     }
 
+    async setUsageAttribution(user: User, usageAttributionId: string | undefined): Promise<void> {
```

Comment:
Here we allow to set it to any team (or user) you're a member of. It doesn't matter if the team is a paying team or not. Is this the desired behaviour?

Reply:
Let me check if there is a simple way to compute that state.

Reply:
checking for `stripeService.findCustomerByTeamId(attributionId.teamId)` now

Reply:
So the usage attribution is set on the non-EE version. That means that we can't assume there will be a subscription. I'm actually okay with the version where we don't check for now. We can always add.